di/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/di/DependencyInjection.java

```diff
@@ -241,18 +239,21 @@ private boolean hasServiceOfType(ServiceDescription<?> serviceDescription) {
 
 	}
 
+	private <T> void typeSaveResolve(@Nonnull ServiceConsumer<T> dependency, @Nonnull Deque<ServiceProvider<?>> stack,
```

Comment:
java doc comments missing, same thing in other methods here

Reply:
this is a private method

Reply:
private methods have to be documented too

Reply:
ok